gensrc/proto/lake_types.proto

```diff
@@ -116,11 +125,17 @@ message TxnLogPB {
         optional DelvecMetadataPB delvec_meta = 4;
     }
 
+    message OpAlterMeta {
+        required TabletMetaTypePB tablet_meta_type = 1;
+        required TabletMetaModifyPB tablet_meta_modfiy = 2;
```

Comment:
How about using the `repeated` type? This would allow better support for modifying multiple metadata attributes at the same time in the future

Reply:
More accurately, it supports modifications with both DELETE and UPDATE operations

Reply:
Ok, I get it